tests/test-splinter-cron.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Cargill Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+version: "3.7"
+
+services:
+
+  unit-test-splinter:
+    build:
+      context: ../
+      dockerfile: tests/Dockerfile
+    image: test-splinter:${ISOLATION_ID}
+    volumes:
+      - ../:/project/splinter
+    command: |
+        bash -c "
+            echo \"Running no-default-features tests...\" && \
```

Comment:
Is there a specific reason we aren't doing this as part of PR builds, or is it just that we're already running the unit tests 3 times for PRs?

Reply:
@ltseeley Yeah, we don't expect this to break often, and we're always on a quest to reduce build times.

If it turns out that it fails frequently, we can move it into to PRs.